app/javascript/hooks/queries/site_settings/useSiteSettingAsync.jsx

```diff
@@ -0,0 +1,12 @@
+import { useQuery } from 'react-query';
+import axios from '../../../helpers/Axios';
+
+export default function useSiteSetting(name) {
```

Comment:
Can we name this `useSiteSettingAsync` to make sure its clear?

Reply:
Good catch, thank you.